src/main/java/Duke.java

```diff
@@ -43,7 +42,9 @@ private void dukeRespondTask(String sampleText , Task userTask ){
         System.out.println(MESSAGE_BOUNDARY);
     }
 
-
+    private Boolean continueChat(String input){
```

Comment:
However, the use of space is not consistent here, some methods have space before curly braces while some do not, like this one